README.md
# tocson
Página web HTML5 + JS + CSS


changelog.md
## Version 1.0.0 (2 sep 2019)
- Initial template
- Bootstrap version 3.3.7

## Version 2.0.0 (21 dec 2020)
Updated
- Bootstrap 4.5.3
- FontAwesome Free 5.13.1
- Slick Slider Added
- Sticky navbar issue solved
- Navbar mobile-menu animation added
- PRODUCTOS style updated
- Google map updated

Removed
- Owl-Carousel
- CONTACTO form php

js/script.js
jQuery(function ($) {
  "use strict";

  /* ----------------------------------------------------------- */
  /*  Fixed header
	/* ----------------------------------------------------------- */
  $(window).on("scroll", function () {
    // fixedHeader on scroll
    function fixedHeader() {
      var headerTopBar = $(".top-bar").outerHeight();
      var headerOneTopSpace = $(".header-one .logo-area").outerHeight();

      var headerOneELement = $(".header-one .site-navigation");
      var headerTwoELement = $(".header-two .site-navigation");

      if ($(window).scrollTop() > headerTopBar + headerOneTopSpace) {
        $(headerOneELement).addClass("navbar-fixed");
        $(".header-one").css("margin-bottom", headerOneELement.outerHeight());
      } else {
        $(headerOneELement).removeClass("navbar-fixed");
        $(".header-one").css("margin-bottom", 0);
      }
      if ($(window).scrollTop() > headerTopBar) {
        $(headerTwoELement).addClass("navbar-fixed");
        $(".header-two").css("margin-bottom", headerTwoELement.outerHeight());
      } else {
        $(headerTwoELement).removeClass("navbar-fixed");
        $(".header-two").css("margin-bottom", 0);
      }
    }
    fixedHeader();

    // Count Up
    function counter() {
      var oTop;
      if ($(".counterUp").length !== 0) {
        oTop = $(".counterUp").offset().top - window.innerHeight;
      }
      if ($(window).scrollTop() > oTop) {
        $(".counterUp").each(function () {
          var $this = $(this),
            countTo = $this.attr("data-count");
          $({
            countNum: $this.text(),
          }).animate(
            {
              countNum: countTo,
            },
            {
              duration: 1000,
              easing: "swing",
              step: function () {
                $this.text(Math.floor(this.countNum));
              },
              complete: function () {
                $this.text(this.countNum);
              },
            }
          );
        });
      }
    }
    counter();

    // scroll to top btn show/hide
    function scrollTopBtn() {
      var scrollToTop = $("#back-to-top"),
        scroll = $(window).scrollTop();
      if (scroll >= 50) {
        scrollToTop.fadeIn();
      } else {
        scrollToTop.fadeOut();
      }
    }
    scrollTopBtn();
  });

  $(document).ready(function () {
    // navSearch show/hide
    function navSearch() {
      $(".nav-search").on("click", function () {
        $(".search-block").fadeIn(350);
      });
      $(".search-close").on("click", function () {
        $(".search-block").fadeOut(350);
      });
    }
    navSearch();

    // navbarDropdown
    function navbarDropdown() {
      if ($(window).width() < 992) {
        $(".site-navigation .dropdown-toggle").on("click", function () {
          $(this).siblings(".dropdown-menu").animate(
            {
              height: "toggle",
            },
            300
          );
        });

        var navbarHeight = $(".site-navigation").outerHeight();
        $(".site-navigation .navbar-collapse").css(
          "max-height",
          "calc(100vh - " + navbarHeight + "px)"
        );
      }
    }
    navbarDropdown();

    // back to top
    function backToTop() {
      $("#back-to-top").on("click", function () {
        $("#back-to-top").tooltip("hide");
        $("body,html").animate(
          {
            scrollTop: 0,
          },
          800
        );
        return false;
      });
    }
    backToTop();

    // banner-carousel
    function bannerCarouselOne() {
      $(".banner-carousel.banner-carousel-1").slick({
        slidesToShow: 1,
        slidesToScroll: 1,
        autoplay: true,
        dots: true,
        speed: 600,
        arrows: true,
        prevArrow:
          '<button type="button" class="carousel-control left" aria-label="carousel-control"><i class="fas fa-chevron-left"></i></button>',
        nextArrow:
          '<button type="button" class="carousel-control right" aria-label="carousel-control"><i class="fas fa-chevron-right"></i></button>',
      });
      $(".banner-carousel.banner-carousel-1").slickAnimation();
    }
    bannerCarouselOne();

    // banner Carousel Two
    function bannerCarouselTwo() {
      $(".banner-carousel.banner-carousel-2").slick({
        fade: true,
        slidesToShow: 1,
        slidesToScroll: 1,
        autoplay: true,
        dots: false,
        speed: 600,
        arrows: true,
        prevArrow:
          '<button type="button" class="carousel-control left" aria-label="carousel-control"><i class="fas fa-chevron-left"></i></button>',
        nextArrow:
          '<button type="button" class="carousel-control right" aria-label="carousel-control"><i class="fas fa-chevron-right"></i></button>',
      });
    }
    bannerCarouselTwo();

    // pageSlider
    function pageSlider() {
      $(".page-slider").slick({
        fade: true,
        slidesToShow: 1,
        slidesToScroll: 1,
        autoplay: true,
        dots: false,
        speed: 600,
        arrows: true,
        prevArrow:
          '<button type="button" class="carousel-control left" aria-label="carousel-control"><i class="fas fa-chevron-left"></i></button>',
        nextArrow:
          '<button type="button" class="carousel-control right" aria-label="carousel-control"><i class="fas fa-chevron-right"></i></button>',
      });
    }
    pageSlider();

    // Shuffle js filter and masonry
    function PRODUCTOShuffle() {
      if ($(".shuffle-wrapper").length !== 0) {
        var Shuffle = window.Shuffle;
        var myShuffle = new Shuffle(
          document.querySelector(".shuffle-wrapper"),
          {
            itemSelector: ".shuffle-item",
            sizer: ".shuffle-sizer",
            buffer: 1,
          }
        );
        $('input[name="shuffle-filter"]').on("change", function (evt) {
          var input = evt.currentTarget;
          if (input.checked) {
            myShuffle.filter(input.value);
          }
        });
        $(".shuffle-btn-group label").on("click", function () {
          $(".shuffle-btn-group label").removeClass("active");
          $(this).addClass("active");
        });
      }
    }
    //PRODUCTOShuffle();

    // testimonial carousel
    function testimonialCarousel() {
      $(".testimonial-slide").slick({
        slidesToShow: 1,
        slidesToScroll: 1,
        dots: true,
        speed: 600,
        arrows: false,
      });
    }
    testimonialCarousel();

    // team carousel
    function teamCarousel() {
      $(".team-slide").slick({
        dots: false,
        infinite: false,
        speed: 300,
        slidesToShow: 4,
        slidesToScroll: 2,
        arrows: true,
        prevArrow:
          '<button type="button" class="carousel-control left" aria-label="carousel-control"><i class="fas fa-chevron-left"></i></button>',
        nextArrow:
          '<button type="button" class="carousel-control right" aria-label="carousel-control"><i class="fas fa-chevron-right"></i></button>',
        responsive: [
          {
            breakpoint: 992,
            settings: {
              slidesToShow: 3,
              slidesToScroll: 3,
            },
          },
          {
            breakpoint: 768,
            settings: {
              slidesToShow: 2,
              slidesToScroll: 2,
            },
          },
          {
            breakpoint: 481,
            settings: {
              slidesToShow: 1,
              slidesToScroll: 1,
            },
          },
        ],
      });
    }
    teamCarousel();

    // media popup
    function mediaPopup() {
      $(".gallery-popup").colorbox({
        rel: "gallery-popup",
        transition: "slideshow",
        innerHeight: "500",
      });
      $(".popup").colorbox({
        iframe: true,
        innerWidth: 600,
        innerHeight: 400,
      });
    }
    mediaPopup();
  });
});






js/featured_products.js
//bd products
const data_products = [
  {
    name: "PANEL DIRECCIONABLE 1 LAZO 60 PUNTOS -CERTIF. UL TEC",
    model: "FX-350-60-DR",
    brand: "MIRCOM",
    category: "PANELES",
    image: "images/products/FX-350-60-DR.svg",
  }, 
  {
    name: "DETECTOR DE HUMO FOTOELECTRICO Y TEMPERATURA DE 04 HILOS",
    model: "SD119-4H-12",
    brand: "SENTEK",
    category: "DETECTORES",
    image: "images/products/SD119-4H-12.svg",
  },
  {
    name: "SIRENA CON LUZ ESTROBO MULTICANDELA RED",
    model: "SYS-HS",
    brand: "System Sensor",
    category: "SIRENAS",
    image: "images/products/SYS-HS.svg",
  },
  {
    name: "ESTACION MANUAL SIMPLE ACCION",
    model: "MS-401U",
    brand: "MIRCOM",
    category: "MANUALES",
    image: "images/products/MS-401U.svg",
  },
];

const productFilter = (categoria = "", marca = "") => {
  let productos = [];

  //busca por la categoria
  if (categoria.length > 0 && marca.length === 0) {
    productos = data_products.filter(
      (product) => product.category == categoria
    );
    //busca por la marca
  } else if (marca.length > 0 && categoria.length === 0) {
    productos = data_products.filter((product) => product.brand == marca);
    //busca por la vategoria y la marca
  } else if (marca.length > 0 && categoria.length > 0) {
    productos = data_products.filter(
      (product) => product.brand == marca && product.category == categoria
    );
  } else {
    //busca todos los productos
    productos = data_products.map((product) => product);
  }
  return productos;
};

function crearCards(categoria = "", marca = "") {
  //se define donde se colocará
  const card = document.querySelector("#card1");

  //se crea el template
  const template = document.querySelector("#template").content;

  //se crea el fragment
  const fragment = document.createDocumentFragment();

  //se obtiene la lista de productos
  const listaProductos = productFilter(categoria, marca);

  //se recorre la lista de productos
  listaProductos.forEach((product) => {
    template.querySelector(".service-box-title").textContent = product.name;
    template.querySelector(".service-box-title").textContent = product.name;

    template.querySelector(".card-subtitle").textContent = product.brand;
    // template.querySelector('.card-text').textContent = product.brand;
    template
      .querySelector(".card-img-top")
      .setAttribute("src", `${product.image}`);

    template.querySelector(".modelo").textContent = product.model;

    template
      .querySelector(".btnlink")
      .setAttribute(
        "href",
        `https://api.whatsapp.com/send?phone=960904097&text=%C2%A1Hola!%20Estoy%20interesado%20en%20cotizar%20el%20producto: ${product.name}%20`
      );

    const clone = template.cloneNode(true);
    fragment.appendChild(clone);
  });

  card.appendChild(fragment);
}

crearCards();

plugins/google-map/map.js
window.marker = null;

function initialize() {
  var map;
  var latitude = $('#map').attr('data-latitude');
  var longitude = $('#map').attr('data-longitude');
  var mapMarker = $('#map').attr('data-marker');
  var mapMarkerName = $('#map').attr('data-marker-name');
  var nottingham = new google.maps.LatLng(latitude, longitude);
  var style = [{
      "featureType": "administrative.locality",
      "elementType": "all",
      "stylers": [{
          "hue": "#2c2e33"
        },
        {
          "saturation": 7
        },
        {
          "lightness": 19
        },
        {
          "visibility": "on"
        }
      ]
    },
    {
      "featureType": "administrative.locality",
      "elementType": "labels.text",
      "stylers": [{
          "visibility": "on"
        },
        {
          "saturation": "-3"
        }
      ]
    },
    {
      "featureType": "administrative.locality",
      "elementType": "labels.text.fill",
      "stylers": [{
        "color": "#282a00"
      }]
    },
    {
      "featureType": "landscape",
      "elementType": "all",
      "stylers": [{
          "hue": "#ffffff"
        },
        {
          "saturation": -100
        },
        {
          "lightness": 100
        },
        {
          "visibility": "simplified"
        }
      ]
    },
    {
      "featureType": "poi",
      "elementType": "all",
      "stylers": [{
          "hue": "#ffffff"
        },
        {
          "saturation": -100
        },
        {
          "lightness": 100
        },
        {
          "visibility": "off"
        }
      ]
    },
    {
      "featureType": "poi.school",
      "elementType": "geometry.fill",
      "stylers": [{
          "color": "#f39247"
        },
        {
          "saturation": "0"
        },
        {
          "visibility": "on"
        }
      ]
    },
    {
      "featureType": "road",
      "elementType": "geometry",
      "stylers": [{
          "hue": "#DB1616"
        },
        {
          "saturation": "100"
        },
        {
          "lightness": 31
        },
        {
          "visibility": "simplified"
        }
      ]
    },
    {
      "featureType": "road",
      "elementType": "geometry.stroke",
      "stylers": [{
          "color": "#DB1616"
        },
        {
          "saturation": "0"
        }
      ]
    },
    {
      "featureType": "road",
      "elementType": "labels",
      "stylers": [{
          "hue": "#008eff"
        },
        {
          "saturation": -93
        },
        {
          "lightness": 31
        },
        {
          "visibility": "on"
        }
      ]
    },
    {
      "featureType": "road.arterial",
      "elementType": "geometry.stroke",
      "stylers": [{
          "visibility": "on"
        },
        {
          "color": "#f3dbc8"
        },
        {
          "saturation": "0"
        }
      ]
    },
    {
      "featureType": "road.arterial",
      "elementType": "labels",
      "stylers": [{
          "hue": "#bbc0c4"
        },
        {
          "saturation": -93
        },
        {
          "lightness": -2
        },
        {
          "visibility": "simplified"
        }
      ]
    },
    {
      "featureType": "road.arterial",
      "elementType": "labels.text",
      "stylers": [{
        "visibility": "off"
      }]
    },
    {
      "featureType": "road.local",
      "elementType": "geometry",
      "stylers": [{
          "hue": "#e9ebed"
        },
        {
          "saturation": -90
        },
        {
          "lightness": -8
        },
        {
          "visibility": "simplified"
        }
      ]
    },
    {
      "featureType": "transit",
      "elementType": "all",
      "stylers": [{
          "hue": "#e9ebed"
        },
        {
          "saturation": 10
        },
        {
          "lightness": 69
        },
        {
          "visibility": "on"
        }
      ]
    },
    {
      "featureType": "water",
      "elementType": "all",
      "stylers": [{
          "hue": "#e9ebed"
        },
        {
          "saturation": -78
        },
        {
          "lightness": 67
        },
        {
          "visibility": "simplified"
        }
      ]
    }
  ];
  var mapOptions = {
    center: nottingham,
    mapTypeId: google.maps.MapTypeId.ROADMAP,
    backgroundColor: "#000",
    zoom: 10,
    panControl: !1,
    zoomControl: !0,
    mapTypeControl: !1,
    scaleControl: !1,
    streetViewControl: !1,
    overviewMapControl: !1,
    zoomControlOptions: {
      style: google.maps.ZoomControlStyle.LARGE
    }
  }
  map = new google.maps.Map(document.getElementById('map'), mapOptions);
  var mapType = new google.maps.StyledMapType(style, {
    name: "Grayscale"
  });
  map.mapTypes.set('grey', mapType);
  map.setMapTypeId('grey');
  var marker_image = mapMarker;
  var pinIcon = new google.maps.MarkerImage(marker_image, null, null, null, new google.maps.Size(46, 40));
  marker = new google.maps.Marker({
    position: nottingham,
    map: map,
    icon: pinIcon,
    title: mapMarkerName
  })
}
var map = document.getElementById('map');
if (map != null) {
  google.maps.event.addDomListener(window, 'load', initialize)
}

js/load_products.js
const data_products = [
    {
      id: 1,
      name: "PANEL DIRECCIONABLE 1 LAZO 60 PUNTOS -CERTIF. UL TEC",
      model: "FX-350-60-DR",
      brand: "MIRCOM",
      category: "PANELES",
      image: "images/products/FX-350-60-DR.svg",
    },
    {
      id: 2,
      name: "PANEL CONVENCIONAL DE 4 ZONAS CONTROL ALARMA DE INCENDIO",
      model: "RE-2554",
      brand: "RAVEL",
      category: "PANELES",
      image: "images/products/RE-2554.svg",
    },
    {
      id: 3,
      name: "BATERIA PS7-12",
      model: "PS7-12",
      brand: "NOVA",
      category: "BATERIAS",
      image: "images/products/PS7-12.svg",
    },
    {
      id: 4,
      name: "CABLE FPL 4X22 LSZH 305 M",
      model: "9013",
      brand: "Dixon",
      category: "CABLES",
      image: "images/products/9013.svg",
    },
    {
      id: 5,
      name: "CABLE UL-FPLR 16 AWG",
      model: "AWGUSH162C",
      brand: "NHT",
      category: "CABLES",
      image: "images/products/AWGUSH162C-AWGUSH182C.svg",
    },
    {
      id: 6,
      name: "CABLE UL-FPLR 18 AWG",
      model: "AWGUSH182C",
      brand: "NHT",
      category: "CABLES",
      image: "images/products/AWGUSH162C-AWGUSH182C.svg",
    },
    {
      id: 7,
      name: "DETECTOR DE HUMO FOTOELECTRICO Y TEMPERATURA DE 04 HILOS",
      model: "SD119-4H-12",
      brand: "SENTEK",
      category: "DETECTORES",
      image: "images/products/SD119-4H-12.svg",
    },
    { 
      id:8,
      name: "DETECTOR DE HUMO DIRECCIONABLE 1 DIRECCION – CERTIFICACION UL",
      model: "MIX-3100",
      brand: "MIRCOM",
      category: "DETECTORES",
      image: "images/products/MIX-3100.svg",
    },
    {
      id:9,
      name: "BASE ESTANDAR PARA DIRECCIONABLE SIN BORDE",
      model: "MIX-2000",
      brand: "MIRCOM",
      category: "DETECTORES",
      image: "images/products/MIX-2000.svg",
    },
    {
      id:10,
      name: "SIRENA CON LUZ ESTROBO MULTICANDELA RED",
      model: "SYS-HS",
      brand: "System Sensor",
      category: "SIRENAS",
      image: "images/products/SYS-HS.svg",
    },
    {
      id:11,
      name: "SIRENA CON ESTROBO NO DIRECCIONABLE PARED ROJO",
      model: "4906-9127",
      brand: "SIMPLEX",
      category: "SIRENAS",
      image: "images/products/4906-9127.svg",
    },
    {
      id:12,
      name: "SIRENA CON LUZ ESTROBO PARA EXTERIOR",
      model: "P2RK",
      brand: "System Sensor",
      category: "SIRENAS",
      image: "images/products/P2RK.svg",
    },
    {
      id:13,
      name: "ESTACION MANUAL SIMPLE ACCION",
      model: "MS-401U",
      brand: "MIRCOM",
      category: "MANUALES",
      image: "images/products/MS-401U.svg",
    },
    {
      id:14,
      name: "ESTACION MANUAL DIRECCIONABLE DOBLE ACCION - CERTIF. UL TEC",
      model: "MS-710IDU",
      brand: "MIRCOM",
      category: "MANUALES",
      image: "images/products/MS-710IDU.svg",
    },
    {
      id:15,
      name: "EXPANSOR DE LAZO DIRECCIONABLE TEC",
      model: "ALCN-252",
      brand: "MIRCOM",
      category: "EXPANSOR",
      image: "images/products/ALCN-252.svg",
    },
  ];
  
  const productFilter = (categoria = "", marca = "") => {
    let productos = [];
  
    //busca por la categoria
    if (categoria.length > 0 && marca.length === 0) {
      productos = data_products.filter(
        (product) => product.category == categoria
      );
      //busca por la marca
    } else if (marca.length > 0 && categoria.length === 0) {
      productos = data_products.filter((product) => product.brand == marca);
      //busca por la vategoria y la marca
    } else if (marca.length > 0 && categoria.length > 0) {
      productos = data_products.filter(
        (product) => product.brand == marca && product.category == categoria
      );
    } else {
      //busca todos los productos
      productos = data_products.map((product) => product);
    }
    return productos;
  };
  
  function crearCards(categoria = "", marca = "") {
    //se define donde se colocará
    const card = document.querySelector("#card1");
  
    //se crea el template
    const template = document.querySelector("#template").content;
  
    //se crea el fragment
    const fragment = document.createDocumentFragment();
  
    //se obtiene la lista de productos
    const listaProductos = productFilter(categoria, marca);
  
    //se recorre la lista de productos
    listaProductos.forEach((product) => {
      template.querySelector(".service-box-title").textContent = product.name;
      template.querySelector(".service-box-title").textContent = product.name;
  
      template.querySelector(".card-subtitle").textContent = product.brand;
      // template.querySelector('.card-text').textContent = product.brand;
      template
        .querySelector(".card-img-top")
        .setAttribute("src", `${product.image}`);
  
      template.querySelector(".modelo").textContent = product.model;
  
      template
        .querySelector(".btnlink")
        .setAttribute(
          "href",
          `https://api.whatsapp.com/send?phone=960904097&text=%C2%A1Hola!%20Estoy%20interesado%20en%20cotizar%20el%20producto: ${product.name}%20`
        );
  
      const clone = template.cloneNode(true);
      fragment.appendChild(clone);
    });
  
    card.appendChild(fragment);
  }
 
  // const crearCardsBaterias = () => {
  //   //const listaProductos = productFilter(categoria, marca);
  
  //   //se recorre la lista
  //   data_products.forEach((product) => {
  //     const image = document.createElement("img");
  //     image.src = "images/projects/bateria-ps7-12.svg";
  //     image.className = "card-img-top";
  //     image.alt = "Bateria PS7-12 | TOCSON";
  
  //     const cardBody = document.createElement("div");
  //     cardBody.className = "card-body";
  
  //     const cardTitle = document.createElement("h5");
  //     cardTitle.className = "card-title text-center";
  //     cardTitle.textContent = `${product.name}`;
  
  //     const parrafo = document.createElement("p");
  //     parrafo.className = "card-text text-center";
  //     parrafo.textContent = `${product.category}`;
  
  //     const enlace = document.createElement("a");
  //     enlace.className = "btn btn-primary w-100";
  //     enlace.textContent = "Cotiza";
  
  //     cardBody.appendChild(cardTitle);
  //     cardBody.appendChild(parrafo);
  //     cardBody.appendChild(enlace);
  
  //     //dónde lo vas a colocar
  //     const card = document.querySelector("#card1");
  //     card.appendChild(image);
  //     card.appendChild(cardBody);
  //   });
  // };
  
  
  $("#baterias").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    if ($(".card-body").length === 0) {
      crearCards("BATERIAS");
    } else {
      console.log("No puedes agregar más");
    }
  });
  
  $("#sirenas").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards("SIRENAS");
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  
  $("#todos").click(function () {
    //$("#card1").load(" #card1");
    //$('.card-body').removeClass('.card-body');
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards();
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  
  $("#manuales").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards("MANUALES");
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  $("#cables").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards("CABLES");
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  $("#paneles").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards("PANELES");
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  $("#exp_lazo").click(function () {
    $("#card1").empty();
    $("label").removeClass("active");
    $(this).addClass("active");
    //window.location.reload(true);
    if ($(".card-body").length === 0) {
      crearCards("EXPANSOR");
    } else {
      console.log("No puedes agregar más porque ya existe");
    }
  });
  $("#detec_humo").click(function () {
      $("#card1").empty();
      $("label").removeClass("active");
      $(this).addClass("active");
      //window.location.reload(true);
      if ($(".card-body").length === 0) {
        crearCards("DETECTORES");
      } else {
        console.log("No puedes agregar más porque ya existe");
      }
    });
  
  crearCards();